bedrock/firefox/redirects.py

```diff
@@ -540,4 +540,11 @@ def firefox_channel(*args, **kwargs):
 
     # bug 1386112
     redirect(r'^firefox/android/faq/?', 'https://support.mozilla.org/products/mobile'),
+
+    # bug 1392796
+    redirect(r'^firefox/desktop/fast/?', 'firefox.features.fast'),
+    redirect(r'^firefox/desktop/trust/?', 'firefox.features.independent'),
+    redirect(r'^firefox/desktop/tips/?', 'firefox.features.index'),
+    redirect(r'^firefox/desktop/customize/?', 'https://support.mozilla.org/kb/customize-firefox-controls-buttons-and-toolbars'),
+    redirect(r'^firefox/private-browsing/?', 'firefox.features.private-browsing'),
```

Comment:
Do we need to remove the existing redirects for `/en-US/firefox/private-browsing/` above on line 525, and `/en-US/firefox/desktop/fast/` on line 529 above now that we have these newer all-locale redirects? We should probably update the redirect tests to reflect these changes too.

Reply:
Nice spot. I think we're fine just removing the `en-US` only redirects and their tests, as the new redirects/tests cover all locales.

Reply:
Are we missing a redirect test for `/firefox/fast/`?

Reply:
> Are we missing a redirect test for /firefox/fast/?

Was that ever a live URL? 404's on prod currently. There is a redirect & test for `/firefox/desktop/fast/`...

Reply:
Sorry I meant `/firefox/features/fast/` I can only see a test for the en-US redirect

Reply:
I have that test locally - waiting for integration tests to finish/pass before updating the PR. 😉